indico/htdocs/js/indico/jquery/categorynavigator.js

```diff
@@ -28,8 +28,22 @@
             openInDialog: false,
             // The title for the category navigator dialog
             dialogTitle: $T.gettext("Select a category"),
-            // Restrict action to categories with no subcategories
-            selectLeafOnly: false,
+            // Allow action to specific categories
+            actionOn: {
+                categoriesWithSubcategories: {
+                    disabled: false,
+                    message: $T.gettext("You cannot select a category that contains subcategories")
+                },
+                categoriesWithEvents: {
+                    disabled: false,
+                    message: $T.gettext("You cannot select a category that contains events")
+                },
+                self: {
+                    disabled: false,
+                    message: $T.gettext("You cannot select the same category"),
```

Comment:
What does it mean **same**? I know where we come from, which was disabling moving a category to itself, which doesn't make sense. However, the broader sense of this option, specially now that you can simply pass a list of IDs, is to disable the action on an arbitrary set of category for whatever reason, in which case **same** and `self` make little sense.

I would understand using **select** if the workflow were selecting a bunch of categories and then performing the action, but when the button is already stating the action, the user is not selecting (only in his mind), the user effectively performing the operation on click. Stretching the *the user always needs to select* argument, you could say that the user doesn't click on a link but is selecting a link to click on. Far fetched in my opinion.